imgui.h

```diff
@@ -34,6 +34,20 @@
 #define IMGUI_IMPL_API      IMGUI_API
 #endif
 
+// Define custom loader for the implementations
+#ifdef IMGUI_IMPL_OPENGL_LOADER_GL3W
+#pragma message("gl3w defined")
+#endif
+#ifdef IMGUI_IMPL_OPENGL_LOADER_GLAD
+#pragma message("glad defined")
+#endif
+#if !defined(IMGUI_IMPL_OPENGL_LOADER_GL3W) \
+    && !defined(IMGUI_IMPL_OPENGL_LOADER_GLEW) \
+    && !defined(IMGUI_IMPL_OPENGL_LOADER_GLAD) \
+    && !defined(IMGUI_IMPL_OPENGL_LOADER_CUSTOM)
+#define IMGUI_IMPL_OPENGL_LOADER_GL3W
+#endif
+
```

Comment:
This has to be in imgui_impl_opengl3.h and not imgui.h

Reply:
Ok done. Btw should `imgui_impl_opengl3.h` also `#include "imgui.h"`? Since `IMGUI_IMPL_API` is defined in `imgui.h`, it would be a compile error if the include order were reversed in client code.

Reply:
You are right, but since things have been working well without until now so I wouldn’t change it. This is in case in the future we add user-defines that affect the visibility or certain elements of imgui.h (like we do for imgui_internals.h). The implicit order constraint allows us to keep this a possibility without affecting end-user code.